js/ui/file_manager/ui.file_manager.adaptivity.js

```diff
@@ -27,15 +28,42 @@ class FileManagerAdaptivityControl extends Widget {
 
         this._drawer = this._createComponent($drawer, Drawer, {
             opened: true,
-            template: this.option("drawerTemplate")
+            template: this._createDrawerTemplate.bind(this)
         });
     }
 
+    _createDrawerTemplate(container) {
+        this.option("drawerTemplate")(container);
+        const leftElement = container;
+        const rightElement = this.$element().find(".dx-drawer-content");
```

Comment:
I think this is a bad idea to modify the internal rendering of a Drawer widget. Can we use the Drawer's public API or modify only our own render?